### DISCUSSION TITLE
Documentation: How to use custom transformer?

### DISCUSSION THREAD

**Discussion by scottc-netflix** [Igorbek/typescript-plugin-styled-components#443] [Q&A]
The docs show at the very bottom an example of using createStyledComponentsTransformer, but what do I do with it? Right now I just have the JSON in my tsconfig.json. It would also be a smoother experience if you show how to import a given function like that. Thanks!

**Comment by Igorbek** [comment]
@scottc-netflix The documentation in `README.md` file outlines a few ways how to use it. Most likely you need to use it with one of the bundlers, like Webpack or Rollup, and configure through their typescript loaders/plugins. They allow passing transformers to the TS compiler. Native TypeScript compiler CLI `tsc` does not allow to do this and you may need to use something like `ttypescript`, if that;'s your case. You can also find in the documentation how to configure it. Please feel free to ask clarification questions if the documentation is not helpful for any reason.
